src/client.c

```diff
@@ -1016,18 +1016,21 @@ run_recipe_handler (gpointer user_data)
     RecipeData *recipe_data = (RecipeData*)user_data;
     AppData *app_data = recipe_data->app_data;
     const gchar **env = NULL;
+    gchar *command;
 
     // return the xml doc
     xmlBufferPtr buffer = xmlBufferCreate();
     gssize size = xmlNodeDump(buffer, app_data->xml_doc, recipe_data->recipe_node_ptr, 0, 1);
 
+    command = g_strdup_printf ("%s %s -- %s --port %d --stdin",
```

Comment:
You're making port permanent now?

Reply:
> You're making the port permanent now?

No, check https://github.com/beaker-project/restraint/pull/31#discussion_r412719564